blocks/title/title.css

```diff
@@ -0,0 +1,125 @@
+.title {
+    /* The gradient here doesn't look great on desktop, better on mobile. Maybe we can create a slightly different one later */
+    background: var(--gradient);
+    width: 100vw;
+    margin-left: calc(-50vw + 50%);
+    padding: 26px;
+    border-top: 2px solid var(--blue);
+    border-bottom: 2px solid var(--blue);
+    position: relative;
+    margin-top: 80px;
+    margin-bottom: 40px;
+}
+
+.title h2 {
+    line-height: 1;
+    margin: 1px 0 0;
+    overflow-wrap: break-word;
+}
+
+@keyframes rotate-background {
+  from {
+        background-position: center center;
+        transform: rotate(0deg);
+    }
+
+  to {
+        background-position: center center;
+        transform: rotate(360deg);
+    }
+}
+
+.title .title-price {
+    position: absolute;
+    height: 130px;
+    width: 130px;
+    top: -40%;
+    right: 10%;
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    align-items: center;
+    margin: 0;
+    overflow: hidden;
+}
+
+.title .title-price::before {
+    content: '';
+    position: absolute;
+    background-image: url('../../icons/pricetag.svg');
```

Comment:
would prefer to add the svg in the dom directly than rely on a relative path in the css.

Reply:
Fixed, I replaced the old logic with adding the svg element to the dom directly.